### ISSUE TITLE
Add advanced introduction

### ISSUE THREAD

**Issue by olegKusov** [effector/effector#548]
It will be good to have good introduction to effector when user starts reading docs. 
1. How effector works under the hood (with good looking illustrations and examples).
2. What problems solves effector
3. Why certain decisions were made
4. Comparing effector with other state managers. List pros and cons of effector in compare with another libraries.
5. Add animated list of contributors to main page , to show that library is alive and in active development and has good community support.

**Comment by zerobias** [comment]
1. [effector: we need to go deeper](https://dev.to/effector/effector-we-need-to-go-deeper-4geg). I treat this as strongly advanced knowledge. Just look at its size.
    You doesn't need to know about type theory to use typescript, so I assume that deep graph understanding is not required to use effector
2. Agree. Previously we use [some description](https://github.com/effector/effector/blob/de063e2b1d73b19736b58435eb6177bc97f47c4e/README.md#effector-follows-five-basic-principles), but it was boring to death and actually prevents people from learning
3. Which one? There are tons of decisions continuously made, what will be interesting to read?
4. In such comparisons everyone pretends to be the best (just try to find a counterexample), I don't want to be "everyone", I don't want to use [infomercial](https://en.wikipedia.org/wiki/Infomercial) tricks even if they looks tempting at this time.
    A well described list of advantages without mentioning alternatives is enough, users are able to compare what they read with other libraries
5. Main page is already heavy due to company logos (>1Mb), and I don't quite understand how a list of people's avatars will lead to a conclusion about active development? You won't even see the last time they contributed to the project 

I'll be grateful if you would suggest a shallower explanation or write a list of advantages. it would help us a lot

**Comment by olegKusov** [comment]
> 1. [effector: we need to go deeper](https://dev.to/effector/effector-we-need-to-go-deeper-4geg). I treat this as strongly advanced knowledge. Just look at its size.
>    You doesn't need to know about type theory to use typescript, so I assume that deep graph understanding is not required to use effector
> 2. Agree. Previously we use [some description](https://github.com/effector/effector/blob/de063e2b1d73b19736b58435eb6177bc97f47c4e/README.md#effector-follows-five-basic-principles), but it was boring to death and actually prevents people from learning
> 3. Which one? There are tons of decisions continuously made, what will be interesting to read?
> 4. In such comparisons everyone pretends to be the best (just try to find a counterexample), I don't want to be "everyone", I don't want to use [teleshopping](https://en.wikipedia.org/wiki/Infomercial) tricks even if they looks tempting at this time.
>    A well described list of advantages without mentioning alternatives is enough, users are able to compare what they read with other libraries
> 5. Main page is already heavy due to company logos (>1Mb), and I don't quite understand how a list of people's avatars will lead to a conclusion about active development? You won't even see the last time they contributed to the project
> 
> I'll be grateful if you would suggest a shallower under-the-hood explanation or write a list of advantages. It would help us a lot

I agree with all what you said. About advantages I really doesn't have enough skills with different state managers and don't know theory of state management . but for me effector has less boilerplate , declarative way - as result less errors. And ability to use different units as clock and target really make it fun to build hard flow without imperative code.

**Comment by egorguscha** [comment]
@olegKusov  is it still relevant?

**Comment by sergeysova** [comment]
*Misclick, sorry*
